doc/nrf/config_and_build/dfu/nrf70_fw_patch_update.rst

```diff
@@ -0,0 +1,207 @@
+.. _ug_nrf70_fw_patch_update:
+
+nRF70 Series firmware patch update
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide explains the available option for updating the nRF70 Series firmware patches that reside in the external memory using the Device Firmware Updates (DFU) procedure.
+
+.. note::
+    External memory refers to the memory that is outside the System-on-Chip (SoC), for example, an external flash memory chip, or an external nonvolatile memory (NVM) chip.
+
+Overview
+========
+
+By default, the patch is located in the on-chip memory as part of the nRF Wi-Fi driver code.
+However, you have the option to store it in external memory, which frees up space in the on-chip memory.
+If the patch is situated outside the SoC memory space, it will not be updated automatically during a regular firmware update of the application core.
+In such cases, you must enable the DFU procedure specifically for the nRF70 Series firmware patch on your device.
+
+To learn more about the firmware patch located in the external memory, see the :ref:`ug_nrf70_developing_fw_patch_ext_flash` guide.
+
+Prerequisites
+=============
+
+To use this feature, ensure that the following prerequisites are met:
+
+* The external memory is available and properly configured in the device tree.
+* The external memory has sufficient capacity to accommodate the firmware patches.
+  This includes additional space for potential patch upgrades, such as those required for DFU.
+  The combined size of all firmware patches should not exceed 128 kB.
+* MCUboot is enabled, and the :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` Kconfig option is set to ``y``.
+
+Supported platforms
+===================
+
+The following platforms are supported:
+
+* nRF5340 DK with nRF7002 EK as a shield
+
+Preparing the application
+=========================
+
+The DFU procedure for the nRF70 Series firmware patch, when located in external memory, requires the creation of additional MCUboot partitions.
+This can be accomplished using the Partition Manager by defining the necessary partition layout in the Partition Manager configuration file specific to your application.
+For instructions on configuring basic partitions with the Partition Manager, refer to the :ref:`partition_manager` documentation.
+
+The instructions in this guide assume you are using the Partition Manager and its associated configuration files.
+
+.. _nrf70_fw_patch_update_mcuboot_partitions:
+
+Defining nRF70 series MCUBoot partitions
```

Comment:
```suggestion
Defining nRF70 Series MCUBoot partitions
```

Reply:
```suggestion
Defining nRF70 Series MCUboot partitions
```
Caught another wrongly capitalized MCUboot thanks to this. :)